.github/workflows/test-sui.yaml

```diff
@@ -35,25 +35,23 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Setup Node.js
-        uses: useblacksmith/setup-node@v5
-        with:
-          node-version: 18.x
-          cache: 'npm'
-
-      - name: Install
-        run: npm ci
-
-      - name: Read SUI_VERSION from JSON
-        id: read-json
+      - name: Prepare version.json file
         run: |
-          SUI_VERSION=$(jq -r '.SUI_VERSION' sui/version.json)
-          echo "SUI_VERSION=${SUI_VERSION}" >> $GITHUB_ENV
+          curl -sL https://raw.githubusercontent.com/axelarnetwork/axelar-cgp-sui/v0.6.0/version.json -o version.json
 
       - name: Setup Sui CLI and install dependencies
-        uses: ./.github/actions/setup-sui
-        with:
-          SUI_VERSION: ${{ env.SUI_VERSION }}
+        uses: axelarnetwork/axelar-cgp-sui/.github/actions/install@v0.6.0
```

Comment:
Avoid hardcoding the version. Get it from package.json/node_modules

Reply:
Dynamic values cannot be used in the `uses` syntax because it is resolved at compile time, not at runtime.

Reply:
hmm, use the latest main then so we don't have to keep bumping this. There's a small chance that the latest main breaks but seems unlikely

Reply:
yeah, that would be better